test/test-CAPI.c

```diff
@@ -118,10 +122,81 @@ int test_parse_to_string_requiring_name()
 
     int status = drafter_parse_blueprint_to(source_without_name, &result, parseOptions, options);
 
-    assert(status != 0);
-    assert(result);
+    assert_true(status != 0);
+    assert_true(result);
+
+    assert_true(strstr(result, expected_without_name) != 0);
+
+    free(result);
+
+    return 0;
+};
+
+const char* source_mson = "# GET /\n+ Response 200 (application/json)\n    + Attributes\n        + name: pavan";
```

Comment:
This is `C` code, which means I couldn't use the fixture loading functions we already had.

Reply:
I could just test that `messageBody` string is contained or not. @kylef Would you prefer it that way? It just doesn't test completely.

Reply:
Might be better approach, otherwise we risk other changes affecting the parse result making us have to alter the fixtures in `expected_without_assets`

Reply:
I think the way to test this feature is
1) integration tests running the `drafter` executable
2) unit tests acting on the C++ API

The C API itself isn't properly tested anyway. If we want to fix that, it's a task in itself; probably one where we need to implement a mocked version of our C++ interface (C API tests should ensure proper dispatch to the C++ interface, really).